docs/API.md

```diff
@@ -137,8 +138,9 @@ logger.child({ a: 'property' }).info('hello child!')
 Creates a child logger, setting all key-value pairs in `bindings` as properties
 in the log lines. All serializers will be applied to the given pair.
 
-Child loggers use the same output stream as the parent and inherit
-the current log level of the parent at the time they are spawned.
+Child loggers use the same output stream as the parent and inherit the current
+log level of the parent at the time they are spawned (in case an object was set
+for the parent's level, then the log level of the matching rule).
```

Comment:
How about: `... are spawned. If an object was used to define the parent's level, then the the child will use the level specified by a matching rule (see discussion below).`